doc/whatsnew/2.13.rst

```diff
@@ -502,3 +502,5 @@ Other Changes
 * Fixed false positive for ``global-variable-not-assigned`` when the ``del`` statement is used
 
   Closes #5333
+
+* Fix type hints in class diagrams generated by pyreverse for class methods and methods returning None.
```

Comment:
Could you add the note for your previous fix as well?

Reply:
@DanielNoord This sentence includes both fixes, do you want me to split it into two bullet points ?

Reply:
Ah shit, I didn't notice that. Hmm, I guess this is fine! Thanks!